CondFormats/CTPPSReadoutObjects/xml/rpix_mapping_220_far_TEST.xml

```diff
@@ -0,0 +1,59 @@
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="10" />
+        </rpix_plane>
+        <rpix_plane id="2">
+          <roc id="0" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="7" />
+          <roc id="1" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="7" />
+          <roc id="2" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="7" />
+          <roc id="3" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="8" />
+          <roc id="4" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="8" />
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="8" />
+        </rpix_plane>
+        <rpix_plane id="3">
+          <roc id="0" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="5" />
+          <roc id="1" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="5" />
+          <roc id="2" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="5" />
+          <roc id="3" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="6" />
+          <roc id="4" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="6" />
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="6" />
+        </rpix_plane>
+        <rpix_plane id="4">
+          <roc id="0" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="3" />
+          <roc id="1" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="3" />
+          <roc id="2" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="3" />
+          <roc id="3" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="4" />
+          <roc id="4" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="4" />
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="4" />
+        </rpix_plane>
+        <rpix_plane id="5">
+          <roc id="0" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="1" />
+          <roc id="1" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="1" />
+          <roc id="2" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="1" />
+          <roc id="3" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="2" />
+          <roc id="4" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="2" />
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="2" />
+        </rpix_plane>
+      </rp_detector_set>
+    </station>
+  </arm>
+
+</top>

```

Comment:
@ghellwig do you mean only *_TEST.xml files or all of them? What exactly you mean with "data repo" (sorry for naive question...)?